rewiremock.d.ts

```diff
@@ -156,18 +156,18 @@ declare module 'rewiremock' {
         append(stubs: any): this;
     }
 
-    type ModuleMock = AnyModuleMock;
-    type ProxyFunction = (r: ModuleMock) => Object;
-    type RequireFunction<T> = () => T;
-    type ImportFunction<T> = () => Promise<T>;
-    type AnyImportFunction<T> = RequireFunction<T> | ImportFunction<T>;
+    export type ModuleMock = AnyModuleMock;
+    export type ProxyFunction = (r: ModuleMock) => Object;
+    export type RequireFunction<T> = () => T;
+    export type ImportFunction<T> = () => Promise<T>;
+    export type AnyImportFunction<T> = RequireFunction<T> | ImportFunction<T>;
 
     /**
      * @name rewiremock
      * @class
      * Proxies imports/require in order to allow overriding dependencies during testing.
      */
-    interface rewiremock {
+    export interface rewiremock {
```

Comment:
Would it be a good step to Capitalize it?

Reply:
we could do that here, yeah

Reply:
OK, I've capitalized this one